interface/ceed-operator.c

```diff
@@ -929,12 +932,6 @@ int CeedOperatorGetFieldByName(CeedOperator op, const char *field_name, CeedOper
       return CEED_ERROR_SUCCESS;
     }
   }
-  // LCOV_EXCL_START
```

Comment:
We still need a `return CEED_ERROR_SUCCESS;` at the end here, right?

Reply:
oh, right